src/s1ap/handlers/s1ap_encoder.c

```diff
@@ -93,15 +93,15 @@ int s1ap_mme_encode_service_rej(
 
     log_msg(LOG_DEBUG, "Encode Serivce Rej");
     pdu.present = S1AP_PDU_PR_initiatingMessage;
-    pdu.choice.initiatingMessage = (InitiatingMessage_t*)malloc(sizeof(InitiatingMessage_t));
+    pdu.choice.initiatingMessage = calloc(sizeof(InitiatingMessage_t), sizeof(uint8_t));
     if(pdu.choice.initiatingMessage == NULL)
```

Comment:
We need debate...I believe let the program crash if memory allocation fails..instead of adding error checks for all the places.

Reply:
Removed the null check here.